node/web/public/scripts/viewModels/PaySuccessVM.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Created by willQian on 2015/12/5.
+ */
+Hilary.scope('heinz').register({
+    name: 'PaymentSuccessVM',
+    dependencies: [],
+    factory: function () {
+        'use strict';
+
+        var PaymentSuccessVM,
+            title;
+
+        //payment details
+        PaymentSuccessVM = function () {
+            title = JSON.parse(localStorage.getItem('cart')).books[0].title;
+            title = title.replace(/\s/g, "+").replace('\'', "\\'");
+            title = '\'http://www.amazon.com/s/ref=nb_sb_ss_c_0_13?url=search-alias%3Dstripbooks&field-keywords=' + title + '&sprefix=' + title + '%2Caps%2C159\'';
+            setTimeout("window.open(" + title + ")", 3000);
```

Comment:
Most browsers frown upon `window.open` these days. You are better off providing the user with a link with a blank target: `<a href="/some/location" target="_blank">Click here to download your book</a>`.